src/cartesian/CartesianAxis.tsx

```diff
@@ -2,8 +2,10 @@
  * @fileOverview Cartesian Axis
  */
 import React, { ReactElement, ReactNode, Component, SVGProps } from 'react';
-import _ from 'lodash';
+
+import isFunction from 'lodash/isFunction';
 import classNames from 'classnames';
+import get from 'lodash/get';
```

Comment:
we _may_ want to alias some of these - `import get as _get` or `import get as lodashGet` because `get` is ambiguous

Reply:
Right, I had to do this for isNumber. This PR so far only does what is required.